src/modules/LanItems.rb

```diff
@@ -1229,7 +1229,7 @@ def BuildBondIndex
       deep_copy(index)
     end
 
-    def startmode_overview
+    def startmode_overview(item_id)
```

Comment:
NTH: Consider adding this to the many other methods which are missing an explanation for item_id / itemId.